src/ripple/protocol/impl/Feature.cpp

```diff
@@ -442,6 +442,7 @@ REGISTER_FEATURE(ExpandedSignerList,            Supported::yes, DefaultVote::no)
 REGISTER_FIX    (fixNFTokenDirV1,               Supported::yes, DefaultVote::no);
 REGISTER_FIX    (fixNFTokenNegOffer,            Supported::yes, DefaultVote::no);
 REGISTER_FEATURE(NonFungibleTokensV1_1,         Supported::yes, DefaultVote::no);
+REGISTER_FEATURE(MemoFeeScaling,                Supported::yes, DefaultVote::yes);
```

Comment:
Why do you think this amendment's defaultVote should be set as `Yes`?

Reply:
I don't think the defaultVote value should be set as `Yes` due to the economical changes that the amendment introduces to performing & implementing business models on-chain (that requires the use of the memos field). With the introduction of the Hooks amendment, federated sidechains and L2 apps (codius/evernode), this would incur additional cost to participants as these concepts are utilized at scale.

Reply:
I agree that the amendment should be "default no" and I think that that's probably the direction for almost all new feature amendments. Certainly the fact that adoption of this amendment would require libraries etc to be updated suggests that "no" is the safer option.

But I don't think that the argument that there are things that use memos is an argument _against_ increasing the fees for transactions with memos. The fact is that memos impose a load on the network: they increase the size of the transactions and increase the amount of storage that full history servers have to carry. It's certainly not unreasonable to charge a fee for that increased load.

Reply:
I'm not against the amendment: In fact, I want it to pass voting, considering the load that the XRPL will experience in the future. This change will affect on-chain businesses/apps that actively use memos, this should be manually voted by validators. 💯 Agree with your statement.